bridge/github/import.go

```diff
@@ -587,8 +389,10 @@ func (gi *githubImporter) ensureCommentEdit(repo *cache.RepoCache, b *cache.BugC
 	switch {
 	case edit.DeletedAt != nil:
 		// comment deletion, not supported yet
+		fmt.Println("comment deletion ....")
```

Comment:
if you want to print something, you should say explicitely that it didn't work: `comment deletion is not supported yet`